client/src/views/asset.js

```diff
@@ -45,208 +47,54 @@ export default ({ t, asset, assetTxs, goAsset, openTx, spends, tipHeight, loadin
 
   return layout(
     <div>
-      <div className="jumbotron jumbotron-fluid asset-page">
+      <div className="container back-nav">
+        <a href="/assets/" className="back-link"><span>{`<`}</span>{`All Assets`}</a>
```

Comment:
Links shouldn't start with a leading `/`, this will break when esplora is served from a subdirectory and not from the root of the hostname. There's a `<base href>` tag that makes links relative to the base url.

Reply:
``` `All Assets` ``` should use `t` so it'll get translated.

Reply:
Changes made